src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -163,65 +163,30 @@ internal static bool VerifyRestoreOutput(RestoreRequest request, CacheFile cache
             return true;
         }
 
-        private static bool IsPackageOnDisk(ISet<PackageIdentity> packagesChecked, IEnumerable<VersionFolderPathResolver> pathResolvers, LocalPackageFileCache packageFileCache, PackageIdentity identity)
```

Comment:
This method has no referneces in the source code. Hence deleting this method is good idea.